packages/styles/scss/patterns/sub-patterns/pictogram-item/_pictogram-item.scss

```diff
@@ -11,18 +11,43 @@
 @mixin pictogram-item {
   .#{$prefix}--pictogram-item {
     &__row {
-      @include carbon--make-row();
+      @include carbon--make-row;
     }
-    &__pic-col {
-      margin-top: $carbon--spacing-07;
-      @include carbon--make-col-ready();
-      @include carbon--make-col(2, 16);
+
+    &__wrapper {
+      @include carbon--make-col-ready;
+
+      @include carbon--breakpoint('sm') {
+        @include carbon--make-col(4, 4);
+      }
```

Comment:
breakpoint `sm` doesn't do anything, you can just add line `21` to line `19` instead.